packages/mobile/src/screens/trending-screen/TrendingFilterButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { makeStyles } from 'app/styles'
+
+import { Button } from '../profile-screen/Button'
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  headerButton: {
+    height: 24,
+    paddingHorizontal: spacing(3),
+    minWidth: 88,
+    borderRadius: 6,
+    backgroundColor: palette.secondary
+  },
+  headerButtonText: {
+    fontSize: 14,
+    textTransform: 'none'
+  }
+}))
+export const TrendingFilterButton = () => {
+  const styles = useStyles()
+  return (
+    <Button
+      variant='primary'
+      title='All Genres'
```

Comment:
pull title into a const messages probs?

Reply:
was thinking that, but then realized this will come from the genre list eventually, so this is just a placeholder